### PR TITLE
Nazar/feature/create add workers

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: frontend/src/components/WorkersModal/WorkersModal.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <>
+      <Modal isOpen={isOpen} ariaHideApp={false} className={style.modal}>
+        {isLoading ? (
+          <Spinner />
+        ) : (
+          <div className={style.modalContent}>
+            <button className={style.btnCloseModal} onClick={closeModal}>
+              ✕
+            </button>
+            {!selectWorker || data.length <= 0 ? (
```
Comment (Taraslysak) [inline frontend/src/components/WorkersModal/WorkersModal.tsx:51]: `data.length < 1` ?

File: frontend/src/pages/Home/components/Orders/Order.module.css [hunk 2]
```diff
@@ -58,13 +58,20 @@
...
     border-block-end: 1px solid #000000;
 }
 
+.orderContentNoteWrapText {
+    text-overflow: ellipsis;
+    overflow: hidden;
+}
+
+
+
```
Comment (Taraslysak) [inline frontend/src/pages/Home/components/Orders/Order.module.css:75]: redundant spaces

File: frontend/src/pages/Home/components/Orders/Order.tsx [hunk 3]
```diff
@@ -94,16 +89,20 @@ const Order = ({
...
-      order_id: id,
-      body: {
-        new_status: new_status,
-        frontend_url: window.location.origin,
-      },
-    };
+    const newStatus = buttonsNameByStatus[currentStatusIndex + 1].key;
 
-    changeStatusOrder.mutate(reqData);
+    if (newStatus === OrderStatus.in_progress) {
```
Comment (Taraslysak) [inline frontend/src/pages/Home/components/Orders/Order.tsx:94]:
```
if (newStatus === OrderStatus.in_progress) {
      handlerModal(id, newStatus);
      return
}
```
Try avoiding else statements

**Review by Taraslysak — CHANGES_REQUESTED** [review]
Please, consider minor refactoring

**Review by kipod — APPROVED** [review]
OK. Thank you

**Pull request merged**